File: Claude3-5 sonnet/python/2-3.py
def file_read_write():
    input_file_path = "input.txt"
    output_file_path = "output.txt"

    try:
        # Read from the input file
        with open(input_file_path, "r") as input_file:
            content = input_file.read()

        # Write to the output file
        with open(output_file_path, "w") as output_file:
            output_file.write(content)

        print(f"File content copied from {input_file_path} to {output_file_path}")
    except FileNotFoundError:
        print(f"Error: {input_file_path} not found.")
    except IOError:
        print("An error occurred while reading or writing the file.")

file_read_write()

File: Claude3-5 sonnet/C++/2-3.c
#include <iostream>
#include <fstream>
#include <string>
using namespace std;

int main() {
    string inputFilePath = "input.txt";
    string outputFilePath = "output.txt";
    string content;

    // Read from the input file
    ifstream inputFile(inputFilePath);
    if (inputFile.is_open()) {
        string line;
        while (getline(inputFile, line)) {
            content += line + "\n";
        }
        inputFile.close();
    } else {
        cout << "Unable to open input file." << endl;
        return 1;
    }

    // Write to the output file
    ofstream outputFile(outputFilePath);
    if (outputFile.is_open()) {
        outputFile << content;
        outputFile.close();
        cout << "File content copied from " << inputFilePath << " to " << outputFilePath << endl;
    } else {
        cout << "Unable to open output file." << endl;
        return 1;
    }

    return 0;
}

